src/App.js

```diff
@@ -35,7 +42,23 @@ class App extends Component {
         {console.log(this.state.movies[0])}
         <NavBar />
         <main>
-          <MovieContainer movieArray={this.state.movies} />
+          <Modal
+            isOpen={this.state.selectedMovie !== null}
+            onRequestClose={this.unselectMovie}
+          >
+            {this.state.selectedMovie !== null ? (
+              <MovieModal movie={this.state.selectedMovie} />
+            ) : (
+              false
+            )}
+            <button onClick={this.unselectMovie}>❌</button>
+          </Modal>
+          <MovieContainer
+            movieArray={this.state.movies}
```

Comment:
We should try not to use datatypes in variable names - instead you could name this prop something like `movies`. With it being plural, it implies a collection of movies.